docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,11 @@
 
 ### Review checklist
 
-* The PR solves #INSERT_ISSUE_NUMBER
+* The PR resolves #INSERT_ISSUE_NUMBER
 * All new code is covered with unit tests
 * All new code was formatted with Prettier
 * Linting passes
 * Tests pass
 * Commit messages follow the
-	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
+  [commit guidelines](CONTRIBUTING.md#git-commit-messages)
```

Comment:
Could you actually put this on one line and update prettier so it doesn't change this?